### PR TITLE
fix: update cicd for bot secrets

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -48,6 +48,9 @@ jobs:
       - uses: ansys/actions/doc-deploy-changelog@v8
         with:
           token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
+          bot-user: ${{ secrets.PYANSYS_CI_BOT_USERNAME }}
+          bot-email: ${{ secrets.PYANSYS_CI_BOT_EMAIL }}
+
```
Comment (Revathyvenugopal162) [inline .github/workflows/ci.yml:53]:
```suggestion
```

**Review by franknli — APPROVED** [review]
This will replace my pull request which also fixes accept.txt file.

File: doc/styles/config/vocabularies/ANSYS/accept.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
-(?!)amsys
+(?i)ansys
+(?i)github
+(?)APIs
```
Comment (franknli) [inline doc/styles/config/vocabularies/ANSYS/accept.txt:3]: @Revathyvenugopal162  This line is missing an operator. Did you mean to use (?i)APIs ?

Comment (Revathyvenugopal162) [reply doc/styles/config/vocabularies/ANSYS/accept.txt:3]: ye, good catch

Comment (Revathyvenugopal162) [inline doc/styles/config/vocabularies/ANSYS/accept.txt:3]:
```suggestion
(?i)APIs
```

**Pull request merged**